app/Http/Controllers/NoteController.php

```diff
@@ -62,20 +97,26 @@ public function edit(Note $note)
      * @param Request $request
      * @return string
      */
-    public function update(Request $request)
+    public function update($noteId, Request $request)
```

Comment:
When you are having `$request` parameter, it's good practice, that you always leave it on first place of parameters. Also missing type for `$noteId`